### ISSUE TITLE
Avoiding manual secret variable reset upon deployment

### ISSUE THREAD

**Issue by Resousse** [google-github-actions/deploy-cloud-functions#244]
### TL;DR

For a specific reason, I need to set a secret env variable on a cloud function, manually. But when I deploy the new code of the function via a github action, my manually set env variables are removed.
How to ignore those variable to let them as is?

### Expected behavior

Manually set env variable to be preserved even after an automated deployment

### Observed behavior

Manually set secret env variable are removed after the run of the github actions

### Action YAML

```yaml
- name: Deploy sendEmail
      uses: google-github-actions/deploy-cloud-functions@main
      with:
        name: sendEmailNotification
        runtime: python39
        entry_point: sendEmailNotification
        region: europe-west1
```

### Additional information

I didn't find any info in the doc about this behavior, it's specified for env variable but not secret variables

**Comment by sethvargo** [comment]
Hi @Resousse 

Thank you for opening an issue.

1. (Unrelated) we do not recommend pinning to `@main`. Please consider pinning to `@v0` or a specific version tag instead.

2. This GitHub Action is declarative, meaning you're expected to describe the complete desired state of the Cloud Function deployment and the action will make that state a reality. As such, you would need to specify your secret references as parameters to the action. This is actually true for all inputs. I've tried to make this clearer in #245.

**Comment by Resousse** [comment]
Oops.

Ok, great, thank you :)
